notebooks/01_data_on_manifolds.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<p>is it the orientation of each axis of the bed? Is that why 3 vectors are plotted below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>